sites/platform/src/add-services/postgresql.md

```diff
@@ -666,11 +665,17 @@ extensions not listed here.
 
 {{< note >}}
 
-You can't upgrade to PostgreSQL 12 with the `postgis` extension enabled.
+You can't upgrade to PostgreSQL 12 with the `postgis` extensions enabled.
 For more details, see how to [upgrade to PostgreSQL 12 with `postgis`](#upgrade-to-postgresql-12-with-the-postgis-extension).
 
 {{< /note >}}
 
+{{< note >}}
```

Comment:
I worry, given the length of the extensions section, that adding the note to the bottom will be completely missed by readers. When they get to the `adminpack` extension in the list, the note is still not in the viewport.